src/index.html

```diff
@@ -1,15 +1,156 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form autocomplete="off">
+      <fieldset class="info__section">
+        <legend>Personal information</legend>
+        <p class="field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname">
+        </p>
+        <p class="field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name">
+        </p>
+        <p class="field">
+          <label for="age">How old are You?</label>
+          <input
+            type="numer"
+            id="age"
+            name="age"
+            min="12"
+            max="100"
+            value="12">
+        </p>
+        <p class="field">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth">
+        </p>
+        <p class="field">
+          <label for="rule">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="rule"
+            name="rule"
+            required>
```

Comment:
```suggestion
            required
          >
```
Fix formatting everywhere in your code